content/en/docs/concepts/architecture/mixed-version-proxy.md

```diff
@@ -67,6 +72,22 @@ If these flags are unspecified, peers will use the value from either `--advertis
 `--bind-address` command line argument to the kube-apiserver.
 If those too, are unset, the host's default interface is used.
 
+## Peer-aggregated discovery
+
+When you enable the feature, discovery requests are automatically enabled to serve
+a comprehensive discovery document (listing all resources served by all apiservers in the cluster)
```

Comment:
```suggestion
a comprehensive discovery document (listing all resources served by any apiserver in the cluster)
```

(The prior phrasing could be incorrectly interpreted to be a intersection operation rather than a union operation)